integreat_cms/cms/migrations/0044_alter_user_options.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.16 on 2022-11-06 10:53
+
+from django.db import migrations
+
+
```

Comment:
```suggestion

# pylint: disable=unused-argument
def update_roles(apps, schema_editor):
    """
    Update analytics permissions

    :param apps: The configuration of installed applications
    :type apps: ~django.apps.registry.Apps

    :param schema_editor: The database abstraction layer that creates actual SQL code
    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
    """
    Group = apps.get_model("auth", "Group")
    Permission = apps.get_model("auth", "Permission")


    # Emit post-migrate signal to make sure the Permission objects are created before they can be assigned
    emit_post_migrate_signal(2, False, "default")


    # Clear and update permissions according to new constants
    for role_name in dict(roles.CHOICES):
        group = Group.objects.filter(name=role_name).first()
        # Clear permissions
        group.permissions.clear()
        # Set permissions
        group.permissions.add(
            *Permission.objects.filter(codename__in=roles.PERMISSIONS[role_name])
        )


```
Just changing the constants won't change the permissions on already deployed systems. You always have to re-assign the roles afterwards via a migration.

Reply:
oh right 😅 I've totally forgotten it.